utils/dchemigen/get_image.h

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Needs KOS copyright header.

Reply:
This is missing header where I lifted it from (dcbumpgen).